tests/snuba/api/endpoints/test_organization_group_index.py

```diff
@@ -449,6 +449,20 @@ def test_project(self):
         response = self.get_success_response(query=f"project:{project.slug}")
         assert len(response.data) == 1
 
+    def test_project_negation_invalid(self):
```

Comment:
Could we add a testcase for `query="!project:[valid-project,invalid-project,also-invalid]".

Reply:
yeah, I had one but deleted it haha